src/cf.data.pre

```diff
@@ -4697,13 +4697,16 @@ DOC_START
 	    RESPONSE
 
 		[http::]<Hs	HTTP status code received from the next hop
```

Comment:
If the following is accurate, please use it:

```suggestion
		[http::]<Hs	HTTP status code of the last received virgin reply
```

If it is wrong, please try to find the right definition of this %code using the being-formalized [terminology](https://wiki.squid-cache.org/ResponseTerms).